fr/administration/secure-platform.md

```diff
@@ -0,0 +1,266 @@
+        Options Indexes
+        AllowOverride all
+        Order allow,deny
+        Allow from all
+        Require all granted
+        <IfModule mod_php5.c>
+            php_admin_value engine Off
+        </IfModule>
+
+        RewriteRule ^index\.html$ - [L]
+        RewriteCond %{REQUEST_FILENAME} !-f
+        RewriteCond %{REQUEST_FILENAME} !-d
+        RewriteRule . /index.html [L]
+        ErrorDocument 404 /centreon/index.html
+
+        AddType text/plain hbs
+    </Directory>
+
+    <Directory "/usr/share/centreon/api">
+        Options Indexes
+        AllowOverride all
+        Order allow,deny
+        Allow from all
+        Require all granted
+        <IfModule mod_php5.c>
+            php_admin_value engine Off
+        </IfModule>
+
+        AddType text/plain hbs
+    </Directory>
+</VirtualHost>
+
+RedirectMatch ^/$ /centreon
+```
+
+> N'oubliez pas de changer les directives **SSLCertificateFile** et **SSLCertificateKeyFile** par le chemin d'accès à
+> votre clé et votre certificat.
+
+5. Activez les flags HttpOnly et Secure
+
+Éditez  le fichier **/opt/rh/httpd24/root/etc/httpd/conf/httpd.conf** et ajouter la ligne suivante :
+```apacheconf
+Header always edit Set-Cookie ^(.*)$ $1;HttpOnly;Secure
+```
+
+6. Redémarrez le serveur web Apache pour prendre en compte la configuration
+
+```shell
+systemctl restart httpd24-httpd
+```
+
+Puis vérifiez le statut :
+
+```shell
+systemctl status httpd24-httpd
+```
+
+Si tout est correct, vous devriez avoir quelque chose comme :
+```shel
+● httpd24-httpd.service - The Apache HTTP Server
+   Loaded: loaded (/usr/lib/systemd/system/httpd24-httpd.service; enabled; vendor preset: disabled)
+   Active: active (running) since mar. 2020-05-12 15:39:58 CEST; 25min ago
+  Process: 31762 ExecStop=/opt/rh/httpd24/root/usr/sbin/httpd-scl-wrapper $OPTIONS -k graceful-stop (code=exited, status=0/SUCCESS)
+ Main PID: 31786 (httpd)
+   Status: "Total requests: 850; Idle/Busy workers 50/50;Requests/sec: 0.547; Bytes served/sec: 5.1KB/sec"
+   CGroup: /system.slice/httpd24-httpd.service
+           ├─ 1219 /opt/rh/httpd24/root/usr/sbin/httpd -DFOREGROUND
+           ├─31786 /opt/rh/httpd24/root/usr/sbin/httpd -DFOREGROUND
+           ├─31788 /opt/rh/httpd24/root/usr/sbin/httpd -DFOREGROUND
+           ├─31789 /opt/rh/httpd24/root/usr/sbin/httpd -DFOREGROUND
+           ├─31790 /opt/rh/httpd24/root/usr/sbin/httpd -DFOREGROUND
+           ├─31802 /opt/rh/httpd24/root/usr/sbin/httpd -DFOREGROUND
+           ├─31865 /opt/rh/httpd24/root/usr/sbin/httpd -DFOREGROUND
+           ├─31866 /opt/rh/httpd24/root/usr/sbin/httpd -DFOREGROUND
+           ├─31882 /opt/rh/httpd24/root/usr/sbin/httpd -DFOREGROUND
+           ├─31903 /opt/rh/httpd24/root/usr/sbin/httpd -DFOREGROUND
+           └─32050 /opt/rh/httpd24/root/usr/sbin/httpd -DFOREGROUND
+```
+
+## URI personnalisée
+
+Il est possible de modifier l'URI de Centreon. Par exemple, **/centreon** peut être remplacé par **/monitoring**.
```

Comment:
Il faut préciser qu'au moins un niveau est necessaire on ne peut pas mettre "/" uniquement

Reply:
@sc979 are you sure that 1 level is needed?